vnext/Shared/OInstance.cpp

```diff
@@ -625,35 +619,20 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
 #else
         auto bundleString = JSBigFileString::fromPath(fullBundleFilePath);
 #endif
+        // We can fail this and leave the timestamp at zero. This ends up being
+        // fine, since we will not use cached bytecode if we cannot read the
+        // file or if timestamps don't match. We should move this logic to live
+        // with the rest of the bytecode caching logic.
```

Comment:
Making it so that we no longer pass bundleVersion through JSExecutors is the next thing on my list.